tools/src/main/scala/scala/scalanative/build/Discover.scala

```diff
@@ -89,7 +89,12 @@ object Discover {
 
       (libDirs ++ llvmLibDir).map(s => s"-L$s")
     }
-    libs
+
+    val opts =
```

Comment:
My recommendation would be to put this in `LLVM.scala` in either the compile or link area because you will have much finer control of where the option goes. Also, that portion of the `Discover` is the most problematic area in regards to cross-compiling.

Reply:
Eric,

Thank you for the review & suggestion.

I looked at `LLVM.scala` as you suggested.  Looks like a fine Swiss mechanical watch. I would
hate to drop a blantant hack in there.

`LLVM.scala` also reminded me that I must keep not only binary executable (a.out) files in mind
but also archive (.a) and sharable libraries (.so).   I think the stack attributes need not be specified
for the later two, at least with some linkers. In fact, the command line arguments for stack and 
archive/shareable may conflict (on some architectures, something else which increases the number
of conditions to be considered/tested).

I am going to back off and try an approach which puts the solution closer to the offending ".S" 
assembly files.  See how far I get.